src/components/molecules/Calendar.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { TouchableOpacity, Text, StyleSheet } from 'react-native';
+import { Calendar, LocaleConfig } from 'react-native-calendars';
+import { CalenderDateType } from '../../types';
```

Comment:
```suggestion
import { CalenderDateType } from '../../types';
```
```suggestion
import { CalendarDateType } from '../../types';
```
ここもCalendarDateTypeでは